backend/app/Repositories/EloquentButtonDataPageViews.php

```diff
@@ -7,6 +7,7 @@
 use App\Contracts\Common\DatePeriod;
 use App\Entities\Visit;
 use App\Repositories\Contracts\ButtonDataPageViews;
+use Illuminate\Support\Facades\DB;
 
 class EloquentButtonDataPageViews implements ButtonDataPageViews
```

Comment:
Please, rename repository to ButtonDataPageViewsRepository, at least. And move repository by namespace PageViews